gov2/aurora/scenarios/get_started_clusters.go

```diff
@@ -0,0 +1,384 @@
+		}
+		parameterGroup, err = scenario.dbClusters.GetParameterGroup(parameterGroupName)
+		if err != nil {
+			panic(err)
+		}
+	}
+	log.Printf("Parameter group %v:\n", *parameterGroup.DBParameterGroupFamily)
+	log.Printf("\tName: %v\n", *parameterGroup.DBClusterParameterGroupName)
+	log.Printf("\tARN: %v\n", *parameterGroup.DBClusterParameterGroupArn)
+	log.Printf("\tFamily: %v\n", *parameterGroup.DBParameterGroupFamily)
+	log.Printf("\tDescription: %v\n", *parameterGroup.Description)
+	log.Println(strings.Repeat("-", 88))
+	return parameterGroup
+
+}
+
+// SetUserParameters shows how to get the parameters contained in a custom parameter
+// group and update some of the parameter values in the group.
+func (scenario GetStartedClusters) SetUserParameters(parameterGroupName string) {
+	log.Println("Let's set some parameter values in your parameter group.")
+	dbParameters, err := scenario.dbClusters.GetParameters(parameterGroupName, "")
+	if err != nil {
+		panic(err)
+	}
+	var updateParams []types.Parameter
+	for _, dbParam := range dbParameters {
+		if strings.HasPrefix(*dbParam.ParameterName, "auto_increment") &&
+			dbParam.IsModifiable && *dbParam.DataType == "integer" {
+			log.Printf("The %v parameter is described as:\n\t%v",
+				*dbParam.ParameterName, *dbParam.Description)
+			rangeSplit := strings.Split(*dbParam.AllowedValues, "-")
+			lower, _ := strconv.Atoi(rangeSplit[0])
+			upper, _ := strconv.Atoi(rangeSplit[1])
+			newValue := scenario.questioner.AskInt(
+				fmt.Sprintf("Enter a value between %v and %v:", lower, upper),
+				demotools.InIntRange{Lower: lower, Upper: upper})
+			dbParam.ParameterValue = aws.String(strconv.Itoa(newValue))
+			updateParams = append(updateParams, dbParam)
+		}
+	}
+	err = scenario.dbClusters.UpdateParameters(parameterGroupName, updateParams)
+	if err != nil {
+		panic(err)
+	}
+	log.Println("You can get a list of parameters you've set by specifying a source of 'user'.")
+	userParameters, err := scenario.dbClusters.GetParameters(parameterGroupName, "user")
+	if err != nil {
+		panic(err)
+	}
+	log.Println("Here are the parameters you've set:")
+	for _, param := range userParameters {
+		log.Printf("\t%v: %v\n", *param.ParameterName, *param.ParameterValue)
+	}
+	log.Println(strings.Repeat("-", 88))
+}
+
+// CreateCluster shows how to create an Aurora DB cluster that contains a database
+// of a specified type. The database is also configured to use a custom DB cluster
+// parameter group.
+func (scenario GetStartedClusters) CreateCluster(clusterName string, dbEngine string,
+	dbName string, parameterGroup *types.DBClusterParameterGroup) *types.DBCluster {
+
+	log.Println("Checking for an existing DB cluster.")
+	cluster, err := scenario.dbClusters.GetDbCluster(clusterName)
+	if err != nil {
+		panic(err)
+	}
+	if cluster == nil {
+		adminUsername := scenario.questioner.Ask(
+			"Enter an administrator user name for the database: ", demotools.NotEmpty{})
+		// TODO: Make this a password questioner.
```

Comment:
Is this TODO intended to be left in here?

Reply:
Good catch! Removed it.